pages.tex

```diff
@@ -104,12 +104,13 @@ \section{Counting Paragraphs}\label{s:pages-counting}
 \end{minted}
 
 \begin{aside}{Developer Tools}
-  If you are using the Firefox browser,
-  you can open the developer tools pane by going to the main menu
-  and selecting \texttt{Tools...\ Web\ Developer...\ Toggle\ Tools}.
+  To open developer tools in Firefox,
+  go to the main menu and select \texttt{Tools {\textgreater} Web\ Developer {\textgreater} Toggle\ Tools}.
   A tabbed display will open in the bottom of your page;
   choose \texttt{Console} to view the output of your JavaScript,
   or to write a little bit to run immediately.
+  You can open a similar set of tools from \texttt{View {\textgreater} Developer {\textgreater} JavaScript\ Console} in Chrome,
+  or by using \texttt{Ctrl+Shift+I} in Microsoft Edge on Windows.
```

Comment:
FYI, this keyboard shortcut (i.e., Ctrl+Shift+I) is applicable to both Firefox and Chrome on Windows as well.

Here are my operating versions in case this is not applicable to Windows and browser versions:

- Windows 7
- Chrome Version 75.0.3770.142
- Firefox 68.0.1